src/main/java/com/palantir/gradle/graal/BaseGraalCompileTask.java

```diff
@@ -37,9 +42,10 @@
 import org.gradle.api.tasks.InputFile;
 import org.gradle.api.tasks.InputFiles;
 import org.gradle.api.tasks.OutputFile;
+import org.gradle.process.ExecSpec;
 
 
-public class BaseGraalCompileTask extends DefaultTask {
+public abstract class BaseGraalCompileTask extends DefaultTask {
```

Comment:
Makes sense. I'd defer this to later, because it's a rather big change. There is also `BaseGraalCompileTask#getArchitectureSpecifiedBinaryPath()`.

If `ExtractGraalTask#getArchitectureSpecifiedBinaryPath(String)` should be moved into the inferface, too, it would need to be a bit more global (instead of a field in `BaseGraalCompileTask`).

Reply:
cool :+1: